WordPress/gutenberg
# Post Editor: Flatten components modes directory

This pull request seeks to eliminate directory nesting within `edit-post/components` for modes. This was originally introduced to clarify a distinction between the two modes: text and visual. As this folder will likely never include more folders, though has had the addition of the keyboard shortcuts component, its value is not clear and should be eliminated.

__Testing instructions:__

Verify that build completes successfully, and there are no errors in loading the editor or switching between modes.
